packages/react-native-codegen/src/parsers/typescript/components/events.js

```diff
@@ -255,28 +239,18 @@ function buildEventSchema(
         argument: getEventArgument(argumentProps, name),
       },
     };
-  }
-
-  if (argumentProps === null) {
-    throw new Error(`Unable to determine event arguments for "${name}"`);
-  }
-
-  if (bubblingType === null) {
+  } else if (!argumentProps) {
```

Comment:
what if we invert this logic a bit? Something like:
```
if (!argumentProps) { throw new Error() }
if (!bubblingType) { throw new Error() }
// rest of the logic
```
I find this approach clearer, it is more efficient 'cause it has to perform less checks and it is less verbose. What do you think? .

Reply:
It doesn't work, Flow's incorrect type inferencing will require me to return a nullable type if I write it in this (original) way.

Flow doesn't even make it right if I move these two if from below to above, it will not remove `void` from `void|others` for `bubblingType` and cause the code not compile.

So I think this is the only correct way to do it.